CONTRIBUTING.md

```diff
@@ -36,6 +36,30 @@ The workflow defined above implies that the community is always ready for discus
 
 Please keep this workflow in mind as you read through the document.
 
+## How to Build and Deploy Locally
+
+After creating a fork and cloning the project locally,
+you can follow the steps below to test your changes:
+
+1. Build your changes:
+
+```sh
+make build go-build-local
+```
+
+2. Build and load the Docker image into Kind:
+
+```sh
+make docker-build
+kind load docker-image quay.io/operator-framework/operator-controller:devel
```

Comment:
`make kind-load` should do the same thing

Reply:
If we change above to be `kind create cluster --name operator-controller` instead of only the default `kind create cluster` then it can work. Otherwise, we face:

```
$ make kind-load
docker save quay.io/operator-framework/operator-controller:devel | /Users/camiladeomacedo/go/bin/kind-v0.24.0 load image-archive /dev/stdin --name operator-controller
ERROR: no nodes found for cluster "operator-controller"
make: *** [kind-load] Error 1
```

I will change towards this direction 👍

Reply:
Ah, yes, we change the name of the cluster... hmmm...